docs/md-enhance/src/guide/playground.md

```diff
@@ -13,7 +13,7 @@ The plugin provides you playground support.
 
 @tab TS
 
-```ts {8-33}
+```ts {8-36}
```

Comment:
4 line is added so I guess the highlight lines are not correct?

Reply:
I checked it, the previous doc has add one more line. so I minus it to make it to right place.